### PR TITLE
dev-python/snakemake: version bump 7.32.3

### PR DESCRIPTION
**Pull request by mschubert**
incl. missing dependencies

### CODE REVIEW

File: dev-python/dpath/dpath-2.1.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{4..11} )
```
Comment (AndrewAmmerlaan) [inline dev-python/dpath/dpath-2.1.6.ebuild:7]: please drop unsupported python versions (anything older then 3.10). Could you also test with 3.12? Adding 3.12 now will save work later.

File: dev-python/dpath/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>mschu.dev@gmail.com</email>
+    <name>Michael Schubert</name>
+  </maintainer>
+  <maintainer type="project">
+    <email>python@gentoo.org</email>
```
Comment (AndrewAmmerlaan) [inline dev-python/dpath/metadata.xml:9]: No Gentoo projects in here please (other then the science project).

**Review by AndrewAmmerlaan — COMMENTED** [review]
Looks good, just some minor comments.

**Comment by mschubert** [comment]
All fixed. 3.12 works for `dpath`, but `plac` needs the `asyncore` backport and `yte` depends on `plac`.

**Comment by AndrewAmmerlaan** [comment]
Great, this looks ready to go now. 

Could you rebase the two fixup commits onto the previous commits? We like to have one commit per package per change, this keeps the log clean and readable.

**Comment by mschubert** [comment]
Also done :heavy_check_mark:

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1:

**Pull request closed without merging**